src/fee_token_liquidity.ts

```diff
@@ -123,10 +112,9 @@ export async function placeFeeTokenLiquidityOrders(
     minBuyAmounts, // buyAmounts
     Array(numOrders).fill(sellAmountOwl), // sellAmount
   );
-  log.info(
-    "Placed fee token liquidity orders for the following tokens" +
-      (await Promise.all(
-        validTokenIds.map(async (i) => await tokenInfo.get(i)?.address),
-      )),
+
+  // Question: Does this really need to be an async mapping?
```

Comment:
Do you plan to merge this question?

If the map contains non-async fields only it's probably not needed (I imagine it used to do a smart contract call which did have to be async).

In any case you don't need to await a promise when you return it.